### ISSUE TITLE
Feature request: Option to use the existing `package.json` rather than copying it

### ISSUE THREAD

**Issue by nategreen** [ng-packagr/ng-packagr#1290]
## Type of Issue

```
[?] Bug Report
[X] Feature Request
```

## Description

Duplicating `package.json` during build causes problems with [lerna](https://github.com/lerna/lerna/issues/1282), because of the packages (source package and compiled package) having duplicate names. But, attempting to set the `dest` option to `"."` or `""` (thinking I could have one `package.json` instead of two that way) yields an error...

```
BUILD ERROR
Source and destination must not be the same.
```

...presumably when the generated `package.json` is trying to be copied on top of the existing one. (Not that I would've wanted all the compiled files co-mingling with the source files anyway.)

#### How to Reproduce

1. Set `ng-package.json`'s `dest` key to `""` and set `deleteDestPath` to `false`
2. Compile it, and you should get an error: `Source and destination must not be the same.`

#### Expected Behavior

Basically, I'd like the option to avoid copying `package.json` and instead add/modify all the APF keys within the main/source `package.json`.

Not sure what would be a clear name for such an option...perhaps something like `--useSourcePackage`.

#### Version Information

```
ng-packagr:            4.7.1
@angular/compiler:     7.2.15
rollup:                0.67.4
tsickle:               0.35.0
typescript:            3.2.4
```

_Please include all version numbers that might be relevant, e.g. third-party libraries_

**Comment by nategreen** [comment]
My bad, only saw #1229 after submitting this. Closing.
